src/cdev/cli/__main__.py

```diff
@@ -39,22 +71,58 @@ def wrap_load_and_initialize_project(
     """
 
     def wrapped_caller(*args, **kwargs):
+        if len(args) == 1:
+            dict_args = vars(args[0])
+
+        elif len(args) == 2:
+            dict_args = vars(args[1])
+
+        else:
+            return
+
+        log_level = dict_args.pop(LOG_LEVEL_ARG)
+        output_type = dict_args.pop(OUTPUT_TYPE_ARG)
+
+        _output_manager = _initialize_output_manager(output_type=output_type)
+
         try:
-            load_and_initialize_project(initialize=initialize)
+            _project = load_and_initialize_project(initialize=initialize)
+        except cdev_core_error as e:
+            _output_manager.print_exception(e)
+            return
         except Exception as e:
-            print(
-                f"Could not load (initialize={initialize}) the Cdev Project to call {command}"
-            )
-            print(e)
-            print(traceback.format_exc())
+            _output_manager.print_exception(wrap_base_exception(e))
```

Comment:
I am going to leave it here for now because moving the check inside the function would require expanding the parameter to accept any general exception, whereas now, the function itself is scoped to being used with Cdev Errors. 

Willing to revisit later depending on how much it pops up in the code base going forward.